src/components/Budget/BudgetTarget/__test__/BudgetTarget.test.tsx

```diff
@@ -111,49 +118,48 @@ const mocks = [
 	},
 ];
 
-// beforeEach(() => {
-// 	act(() => {
-// 		dialog = renderApollo(
-// 			<DashboardProvider
-// 				defaultState={{ project: projectDetails, organization: organizationDetail }}
-// 			>
-// 				<NotificationProvider>
-// 					<BudgetTarget
-// 						formAction={FORM_ACTIONS.CREATE}
-// 						open={true}
-// 						handleClose={handleClose}
-// 					/>
-// 				</NotificationProvider>
-// 			</DashboardProvider>,
-// 			{
-// 				mocks,
-// 				addTypename: false,
-// 			}
-// 		);
-// 	});
-// });
+beforeEach(() => {
+	act(() => {
+		dialog = renderApollo(
+			<DashboardProvider
+				defaultState={{ project: projectDetails, organization: organizationDetails }}
+			>
+				<NotificationProvider>
+					<BudgetTarget
+						formAction={FORM_ACTIONS.CREATE}
+						open={true}
+						handleClose={handleClose}
+					/>
+				</NotificationProvider>
+			</DashboardProvider>,
+			{
+				mocks,
+				addTypename: false,
+			}
+		);
+	});
+});
 
-const inputIds = BudgetTargetinputFields;
+const budgetTargetInputIds = BudgetTargetinputFields;
 
 describe("Budget Target Dialog tests", () => {
```

Comment:
Again..what funtionality are we testing here?

Reply:
I have mentioned the names of the input field and the key to access their values from an object called intialFormValue in a seprate file. Then i loop through all the input fields and insert the value and check if the values are modified and the input field has the correct value, then I check if the submit button got enabled, then I check if the request got submitted. All of these test are releated to each other,